webpack.prod.js

```diff
@@ -0,0 +1,6 @@
+const { merge } = require('webpack-merge');
+const common = require('./webpack.common.js');
+
+module.exports = merge(common, {
+	mode: 'production',
+});
```

Comment:
P3) production 환경에서는 style-loader 대신 mini-css-extract-plugin을 사용하는 것도 좋을 것 같습니다.
style-loader의 경우 css를 javascript로 변환해서 style태그 내부에 js코드로 만들어서 적용을 해주는데
mini-css-extract-plug을 사용하면 js가 아니라 별도의 css파일을 만들어줘서 파일 분리가 가능하다고 합니다.

그런데 이렇게 mini-css-extract-plug을 사용하면 빌드 속도가 느려지고 새로고침을 해야 css 변경사항이 적용된다고 해서
빠르게 확인해야하는 dev환경에서는 적합하지 않아서 style-loader를 그대로 사용하고
그렇지 않은 production 환경에서는 style-loader 대신 mini-css-extract-plug를 사용하는 방법도 있다고 합니다

```suggestion
module.exports = merge(common, {
  mode: 'production',
  plugins: [
    new MiniCssExtractPlugin({
      filename: '[name].[contenthash].css',
    }),
  ],
  module: {
    rules: [
      {
        test: /\.css$/i,
        use: [MiniCssExtractPlugin.loader, 'css-loader'],
      },
    ],
  },
});

```

Reply:
오 감사합니다 찾아봐야겠어요!